data/client-side-encryption/fle2-Compact.yml

```diff
@@ -0,0 +1,79 @@
+tests:
+  - description: "Compact works"
+    clientOptions:
+      autoEncryptOpts:
+        kmsProviders:
+          local: {'key': {'$binary': {'base64': 'Mng0NCt4ZHVUYUJCa1kxNkVyNUR1QURhZ2h2UzR2d2RrZzh0cFBwM3R6NmdWMDFBMUN3YkQ5aXRRMkhGRGdQV09wOGVNYUMxT2k3NjZKelhaQmRCZGJkTXVyZG9uSjFk', 'subType': '00'}}}
+    operations:
+      - name: runCommand
+        object: database
+        command_name: compactStructuredEncryptionData
+        arguments:
+          command:
+            compactStructuredEncryptionData: *collection_name
+    expectations:
+      - command_started_event:
+          command:
+            listCollections: 1
+            filter:
+              name: *collection_name
+          command_name: listCollections
+      - command_started_event:
+          command:
+            find: datakeys
+            filter: {
+                  "$or": [
+                      {
+                          "_id": {
+                              "$in": [
+                                {'$binary': {'base64': 'EjRWeBI0mHYSNBI0VniQEg==', 'subType': '04'}},
+                                {'$binary': {'base64': 'q83vqxI0mHYSNBI0VniQEg==', 'subType': '04'}}
+                              ]
+                          }
+                      },
+                      {
+                          "keyAltNames": {
+                              "$in": []
+                          }
+                      }
+                  ]
+              }
+            $db: keyvault
+            readConcern: { level: "majority" }
+          command_name: find
+      - command_started_event:
+          command:
+            compactStructuredEncryptionData: *collection_name
+            compactionTokens: {
+              "encryptedIndexed": {
+                "$binary": {
+                  "base64": "noN+05JsuO1oDg59yypIGj45i+eFH6HOTXOPpeZ//Mk=",
+                  "subType": "00"
+                }
+              },
+             "encryptedUnindexed": {
+                "$binary": {
+                  "base64": "SWO8WEoZ2r2Kx/muQKb7+COizy85nIIUFiHh4K9kcvA=",
+                  "subType": "00"
+                }
+              }
+            }
+          command_name: compactStructuredEncryptionData
+  - description: "Compact errors on an unencrypted client"
+    operations:
+      - name: runCommand
+        object: database
+        command_name: compactStructuredEncryptionData
+        arguments:
+          command:
+            compactStructuredEncryptionData: *collection_name
+        result:
+          errorContains: "'compactStructuredEncryptionData.compactionTokens' is missing"
```

Comment:
Do we care about these new lines?

Reply:
AFAIK no. Though this is copied directly from the specifications repo.